docs/release-notes/v0.30.0.md

```diff
@@ -0,0 +1,79 @@
+## Announcing Radius v0.30.0
+
+Today we're happy to announce the release of Radius v0.30.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+<!-- TALK TO THE PM TEAM ABOUT WHAT HIGHLIGHTS TO ADD HERE -->
+
+## Breaking changes
+
+None
+
+## New contributors
+
+Welcome to our new contributors who have merged their first PR in this release!
+
+* @IvanJosipovic made their first contribution in https://github.com/radius-project/radius/pull/6996
+* @gpltaylor made their first contribution in https://github.com/radius-project/radius/pull/6974
+* @akshayrohilla made their first contribution in https://github.com/radius-project/radius/pull/7083
+* @jhandel made their first contribution in https://github.com/radius-project/radius/pull/7095
```

Comment:
This got merged after the RC was cut. Is this code in 0.30, or do we have a bug in our release note generation script?

Reply:
Looks like a bug in the release notes generation. @IvanJosipovic made their first contribution in https://github.com/radius-project/radius/pull/6996 is not present in this branch (release-0.30)

Reply:
Looks like a release notes generation issue. This is not present in this branch:-
* @jhandel made their first contribution in https://github.com/radius-project/radius/pull/7095

Reply:
Created: https://github.com/radius-project/radius/issues/7107 to address this